core/ui/ViewTemplate/subtitle.tid

```diff
@@ -4,7 +4,11 @@ tags: $:/tags/ViewTemplate
 \whitespace trim
 <$reveal type="nomatch" stateTitle=<<folded-state>> text="hide" tag="div" retain="yes" animate="yes">
 <div class="tc-subtitle">
```

Comment:
I think we can get red if this div, if we add the `class="tc-subtitle"` to the reveal widget. ... I didn't test it. So it's just a guess

Reply:
hmmm, If we do already touch the structure, IMO we should really try to fix it too. 

I think there is enough new code and DOM elements now, that break backwards compatibility anyway.